spec/resources.yml

```diff
@@ -123,7 +123,7 @@ components:
         ##### Text search
 
         If the `search` parameter is provided, narrows results to those pertaining to the search query. Search works across resource IDs and the values of annotations. It weighs results so that those with matching id or a matching value of an annotation called `name` appear first, then those with another matching annotation value, and finally those with a matching  `kind`."
-        operationId: "getResourcesOnAccount"
+        operationId: "showForAccount"
```

Comment:
@izgeri I like the name `showAll` however I think it may present the wrong Idea to the user. If I was to just look at the method names I would assume that `showAll` would show all resources, and I wouldn't expect to have to pass an account parameter. 

I think that the `show` verb here may be extraneous and we could switch to something like `onAccount` in order to shorten it. The use case of which would roughly be:
```python
resources_api = conjur.api.ResourcesApi()
resources = resources_api.on_account('cucumber')
```
I think this syntactically makes sense as well because it immediately tells you what the passed parameter represents and implies by the lack of other filtering args that we are retrieving all resources.

Any thoughts on this?

Reply:
what about `showForAccount` instead of `showAll`, but otherwise using the suggestions as above?

that means:
- `/resources/{account}` is `showForAccount`
- `/resources` is `showForAllAccounts` (still a super uncommon workflow, we don't really support multi-tenancy right now - which means that this response and the `/resources/{account}` response are going to effectively be the same)
- `/resources/{account}/{kind}` is `showForKind`
- `/resources/{account}/{kind}/{identifier}` is `show`

Reply:
I still rather prefer `showAll` for `/resources/{account}`.

Reply:
There are a few comments relating to this change, I'll respond here and resolve the other ones. I updated the names as in your comment above